src/mainloop.cpp

```diff
@@ -54,8 +54,13 @@
 #define EXPOSURE_P_GAIN 100.0f
 #define EXPOSURE_I_GAIN 0.5f
 #define EXPOSURE_D_GAIN 0.5f
+#define GAIN_P_GAIN 50.0f
+#define GAIN_I_GAIN 0.5f
+#define GAIN_D_GAIN 0.5f
 #define EXPOSURE_CHANGE_THRESHOLD 30.0
 #define EXPOSURE_ABOSULUTE_MAX_VALUE 1727
+#define GAIN_CHANGE_THRESHOLD 15.0
+#define GAIN_ABOSULUTE_MAX_VALUE 100 //TODO check
```

Comment:
You want to know the maximum gain that the camera supports? It is 127.

Reply:
fixed. Thanks!